File: ejercicio_19/ejercicio_19.java
import java.util.Scanner;
public class ejercicio_19 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        String resultado = "",letra;
        char caracter;
        System.out.print("Ingresa varias letras o una cadena de letras: ");
        letra = entrada.nextLine();
        for (int i = 0; i < letra.length(); i++) {
            caracter = letra.charAt(i);
            resultado += Character.toUpperCase(caracter);
            for (int j = 0; j < i; j++) {
                resultado += Character.toLowerCase(caracter);
            }
            if (i < letra.length() - 1) {
                resultado += "-";
            }
        }
        System.out.println(resultado);
        entrada.close();
    }   
}

File: ejercicio_19/ejercicio_19.py
letra = input("Ingresa varias letras o una cadena de letras: ")
resultado = ""
for i in range(len(letra)):
    caracter = letra[i]
    resultado += caracter.upper()
    for j in range(i):
        resultado += caracter.lower()
    if i < len(letra) - 1:
        resultado += "-"
print(resultado)
